crates/host/src/wasmbus/mod.rs

```diff
@@ -2621,6 +2607,14 @@ impl Host {
                 let actor = entry.get_mut();
                 let config_changed =
                     &config != actor.handler.config_data.read().await.config_names();
+
+                if actor.image_reference != component_ref {
+                    warn!(
+                        "Requested to scale existing component to a different image reference: {} != {}. The component will be scaled but the image reference will not be updated. If you meant to update this component to a new image ref, use the update command.",
```

Comment:
Should we send this message back in the response to the user or are we too far down a spawned process?

Reply:
Pushing this